### ISSUE TITLE
SC2188: False warning on file redirection

### ISSUE THREAD

**Issue by Akianonymus** [koalaman/shellcheck#1974]
- My shellcheck version: Online
- The rule's wiki page does not already cover this (https://shellcheck.net/wiki/SC2188)
- I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```bash

#!/usr/bin/env bash
printf "%s\n" "$(< "${SUCCESS}" && < "${ERROR}")")"
                 ^-- SC2188: This redirection doesn't have a command. Move to its command (or use 'true' as no-op)
                                   ^-- SC2188: This redirection doesn't have a command. Move to its command (or use 'true' as no-op)

```
ERROR & SUCCESS are filenames.

This is basically an alternative to cat command, I don't see any problem with it.

Refrence: https://github.com/dylanaraps/pure-bash-bible#read-a-file-to-a-string
